default/scripting/buildings/INTERSPECIES_ACADEMY.focs.txt

```diff
@@ -0,0 +1,94 @@
+BuildingType
+    name = "BLD_INTERSPECIES_ACADEMY"
+    description = "BLD_INTERSPECIES_ACADEMY_DESC"
+    buildcost = 30
+    buildtime = 5
+    location = AND [
+        // Homeworld
+        Species
+        Not Contains Building name = "BLD_INTERSPECIES_ACADEMY"
+        OwnedBy empire = Source.Owner
+        // some species dont have preferred focus - else low=20 would be nicer
+        Happiness low = 15
+        Not WithinStarlaneJumps jumps = 3 condition = And [
+            System
+            Contains And [
+                Building name = "BLD_INTERSPECIES_ACADEMY"
+                OwnedBy empire = Source.Owner
+            ]
+        ]
+        Number low = 0 high = 0 condition = And [
+            Building name = "BLD_INTERSPECIES_ACADEMY"
+            Species name = RootCandidate.Species
+            OwnedBy empire = RootCandidate.Owner
+        ]
+        Number low = 0 high = 0 condition = Described description = "Academies of this species already exist in the empire" condition = And [
```

Comment:
Use stringtable references, not hard-coded english in FOCS.

Reply:
If i remember correctly this is to give a better description why you cant build the academy.
Does described work with stringtables? Or is there another way to add a description of that condition?

Reply:
Not looked at it closely, the only current use is a stringtable lookup: [CULTURE_ARCHIVED.focs.txt:7](https://github.com/freeorion/freeorion/blob/master/default/scripting/buildings/CULTURE_ARCHIVES.focs.txt#L7)